config/sync/user.role.authenticated.yml

```diff
@@ -17,4 +17,3 @@ permissions:
   - 'search content'
   - 'skip comment approval'
   - 'use text format basic_html'
-  - 'use text format loggedin_text_filter'
```

Comment:
Uh oh, I think you removed the changes from Balazs... :) Did you export your config, pull the latest changes then import the config before pushing your changes?